docs/guides/postgres/branch/customization/index.md

```
@@ -0,0 +1,536 @@
+| `Branch` is `Failed` with `reason: PostActionFailed` | A post-action container exited non-zero. Courier suspends the Job so the failed Pod and its logs survive — read them with `kubectl logs -n <ns> job/<target>-post-action`, then fix `spec.postActions` and re-apply. The edit triggers a retry. |
+| A post-action Pod fails to start with an exec error | `jobTemplate.spec` has no `command` field, so `args` are passed to the image's entrypoint. For an entrypoint-less image, `args` must start with the binary, e.g. `["sh", "-c", "…"]`. |
+| A post-action succeeded but the branch still shows raw data after a refresh | Each refresh generation runs its own Job (`<target>-post-action-gen<N>`). Check that the latest generation's Job completed — Courier labels the Jobs it owns with the branch name: `kubectl get job -n <ns> -l courier.kubedb.com/branch=<branch>`. |
+| `FRESHNESS` is not advancing on a scheduled branch | Check `status.history` and `status.nextRefreshTime`, and the Courier logs: `kubectl logs -n <ns> deploy/kubedb-courier`. |
+| A cross-namespace branch's PVC stays `Pending` | The mirrored snapshot in the target namespace is not ready yet, or the target StorageClass uses a different CSI driver than the source. Check `kubectl get volumesnapshot -n <target-ns>`. |
+
+For problems with the core branch flow — snapshotting, cloning, provisioning — see the [walkthrough's troubleshooting table](/docs/guides/postgres/branch/same-cluster/index.md#troubleshooting).
+
+## Cleaning up
+
+```bash
+$ kubectl delete branch -n demo --all
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)